src/main/java/dev/datainmotion/airquality/AirQualityApp.java

```diff
@@ -45,50 +43,36 @@ public static void main(String[] args) {
         SpringApplication.run(AirQualityApp.class, args);
     }
 
-    @Autowired
-    private PulsarTemplate<Observation> pulsarTemplate;
+	@Bean
+	public PulsarTopic newTopic(PulsarProperties pulsarProperties) {
+		return PulsarTopic.builder(pulsarProperties.getProducer().getTopicName()).build();
```

Comment:
We have the ability to auto-create topics on startup (add a `PulsarTopic` bean - get a topic created for you).